src/picknik_ur_site_config/objectives/re-tare.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<root BTCPP_format="4" main_tree_to_execute="Re-tare">
+  <!-- ////////// -->
+  <BehaviorTree ID="Re-tare" _description="Zero the force torque sensors to mitigate accumulated drift error" _favorite="true" _hardcoded="false">
```

Comment:
The name `"Re-tare"` alone seems a little lacking in descriptiveness. What about something like "Re-Zero Force/Torque Sensor" or "Re-Zero FTS"?

Same w/ the file name

Reply:
Done, thank you for this feedback, things definitely look better and make more sense with the added description and use of the Subtree ID references.